x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,342 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: (
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <RecordScore score={initialScore} />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>{` -> `}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <RecordScore score={finalScore} />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+    });
+  }
+
+  const impactDetails = [];
+  if (explanation.anomaly_characteristics_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristicsTooltip',
+            {
+              defaultMessage: 'Anomaly characteristics impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyCharacteristics"
+              defaultMessage="Anomaly characteristics impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.anomaly_characteristics_impact} />,
+    });
+  }
+
+  if (explanation.single_bucket_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.singleBucketTooltip',
+            {
+              defaultMessage: 'Single bucket impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.singleBucket"
+              defaultMessage="Single bucket impact"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: <ImpactVisual score={explanation.single_bucket_impact} />,
+    });
+  }
+  if (explanation.multi_bucket_impact !== undefined) {
+    impactDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.multiBucketTooltip',
+            {
+              defaultMessage: 'Multi bucket impact blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.multiBucket"
+              defaultMessage="Multi bucket impact"
```

Comment:
We hyphenate `multi-bucket` in other parts of the UI (chart tooltip, plus expanded table row). @lcawl any thoughts on whether we should use `multi-bucket` or `multi bucket`? 

In the docs we use for example `An indication of how strongly an anomaly is multi bucket or single bucket.` 

Either way, we should be consistent.

<img width="831" alt="image" src="https://user-images.githubusercontent.com/7405507/195554226-d4965815-0115-4747-ab98-f3af07bf4726.png">

Reply:
Unfortunately, it doesn't look like we've got guidance on this in the [writing guide](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide), since both "multi-cloud" and "multifields" are acceptable there.  We likewise have a mix of "multi metric" and "multi-metric" jobs in our docs and UI.

Reply:
I will open a style issue, but I suspect from the existing examples that they'll advise using "multimetric" and "multibucket".